.travis.yml

```diff
@@ -3,9 +3,14 @@ cache:
   directories:
   - node_modules
 node_js:
- - "8"
+- '8'
 after_success:
 - npm run ci
-
 notifications:
   email: false
+
+deploy:
```

Comment:
are we deploying on every test? do we only want to deploy on push to master?

Reply:
Right now. we only want to deploy staging from `develop` and then `master` we deploy for production

Reply:
then you need to update the travis deploy appropriately. It's not doing what you except. You'll want to set it up with `only: develop` for this one.